app/api/dao/user.py

```diff
@@ -146,13 +147,11 @@ def list_users(user_id, is_verified=None):
         """
 
         users_list = UserModel.query.filter(UserModel.id != user_id).all()
-        list_of_users = []
-        if is_verified:
-            for user in users_list:
-                if user.is_email_verified:
-                    list_of_users += [user.json()]
-        else:
-            list_of_users = [user.json() for user in users_list]
+
+        list_of_users = [user.json() for user in filter(
```

Comment:
Search query should be optional so if there is none it should return all users.

Reply:
It is optional. Take a look at the `user.py` in resources, it gets search query using this code:
`request.args.get('search', '')`
if the search query is None it returns an empty string, which is contained in every possible string.
Take a look at my first screenshot. It is labeled "without query" and it returns all 6 users, that i have in my DB. And then take a look at the second screenshot. It has 'N' as search query and it returns only 4 users, as the last 2 do not contain 'N' in their names. (lorem ipsum dolor sit atmet) And it is also contained in tests. 
`test_list_users_api_without_search_query_resource_auth` returns all users as before (this test already existed) and it doesn't have a search query in it. Hope that clarifies your doubts :)